docs/concepts/configuration/pod-priority-preemption.md

```diff
@@ -207,13 +207,14 @@ Node is preempted. Here's an example:
 
 * Pod P is being considered for Node N.
 * Pod Q is running on another Node in the same zone as Node N.
-* Pod P has anti-affinity with Pod Q.
-* There are no other cases of anti-affinity between Pod P and other Pods in the zone.
-* In order to schedule Pod P on Node N, Pod Q should be preempted, but scheduler
-does not perform cross-node preemption. So, Pod P will be deemed unschedulable
-on Node N.
-
-If Pod Q were removed from its Node, the anti-affinity violation would be gone,
+* Pod P has pod affinity with Pod Q.
```

Comment:
The original text is correct. Pod P must have anti-affinity to Q for the example to be correct. This example is trying to make a case where Pod Q must be removed (preempted) first to allow Pod P to be scheduled on a different node than the one running Pod Q.

Reply:
@bsalamat This is an example of cross-node preemption., right? If P is scheduled on N, and Q is running on M, I don't see how anti-affinity between P and Q (i.e., constraint forcing them to run on *different* nodes) prevents P from being scheduled.

Reply:
@glikson You are right about "the same node", but "cross-node preemption" is required when anti-affinity applies to multiple nodes. Affinity/anti-affinty has a topology key, for example, P can have anti-affinity to Q with topology key being "zone". In that case P and Q must not be scheduled in the same zone. Now if N and M are in the same zone, P and Q cannot be scheduled in the same zone.

Reply:
To clarify things, I'd suggest your PR to change this line to:
"Pod P has anti-affinity with Pod Q with topology key of the anti-affinity being "zone".

Reply:
@bsalamat Thanks for the clarification. Makes sense - updated the PR.